SU2_CFD/src/output_structure.cpp

```diff
@@ -3861,7 +3863,10 @@ void COutput::SetRestart(CConfig *config, CGeometry *geometry, CSolver **solver,
   restart_file <<"INITIAL_BCTHRUST= " << config->GetInitial_BCThrust() << endl;
   restart_file <<"DCD_DCL_VALUE= " << config->GetdCD_dCL() << endl;
   if (adjoint) restart_file << "SENS_AOA=" << solver[ADJFLOW_SOL]->GetTotal_Sens_AoA() * PI_NUMBER / 180.0 << endl;
-  restart_file <<"EXT_ITER= " << config->GetExtIter() + config->GetExtIter_OffSet() + 1 << endl;
+  if (dual_time)
+    restart_file <<"EXT_ITER= " << config->GetExtIter() + 1 << endl;
+  else
+    restart_file <<"EXT_ITER= " << config->GetExtIter() + config->GetExtIter_OffSet() + 1 << endl;
   
   restart_file.close();
   
```

Comment:
Let's save this for the next PR. There is something else we can try rather than including all of the periodic halo points (which means we have to adjust all of the restart loaders again)...

Basically, I think we should just make sure that we perform an MPI send/recv immediately after loading the restart file(s) to make sure that the periodic boundaries are initialized correctly before the first iteration. For unsteady, we also need to make sure that the solution gets "pushed back" to time levels N and N-1 after initializing the periodic points too.

I will be testing some things for this with the small case that you shared with me, @arubino. When this passes the tests, we'll merge and close this PR right away.